### DISCUSSION TITLE
Automatic 1111 dark mode

### DISCUSSION THREAD

**Discussion by tirtho888** [TheLastBen/fast-stable-diffusion#2022] [Q&A]
Is it possible to enable dark mode on Chrome browser for this web UI?

**Comment by TheLastBen** [comment]
add --theme dark, or simply by setting your os to dark mode

**Reply by tirtho888** [reply]
Where should I put the --theme dark command??

**Reply by TheLastBen** [reply]
in the last cell of the A1111 notebook with all the arguments that start with --
anyways, I'll add it to the notebook later

**Comment by ackvf** [comment]
Interesting, my A1111 is dark by default, maybe because of my OS setting and a dark preference?

Anyway I am using this neat extension that turns any website to dark mode. What is  great is that I can tweak the brightness, saturation, tone and other stuff, which gives me a more pleasant interface than the default dark.

Dark Reader
https://chrome.google.com/webstore/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh

default dark
![image](https://user-images.githubusercontent.com/2313018/233852903-3c078a3f-6fb3-46ce-98b4-40f7f1d5c57c.png)

dark reader extension
![image](https://user-images.githubusercontent.com/2313018/233852886-d704d43a-be67-4408-96c7-a24b8f632012.png)
